nuxeo-runtime/nuxeo-runtime-metrics/src/main/resources/OSGI-INF/runtime-metrics-service.xml

```diff
@@ -29,6 +29,29 @@
       </metrics>
       </extension>
 
+      Since 9.3, you can filter the list of metrics reported to Graphite. The ones enabled can be
+      defined with the metrics.graphite.allowedMetrics property and the ones ignored with the
+      metrics.graphite.deniedMetrics property.
+
+      About the allowed metrics, if a metric name starts with any item of this list, then it will be reported.
+      The 'ALL' value means that every metrics will be reported.
+      About the denied metrics, if a metric name starts with any item of this list, then it will be ignored
+      (unless it is part of the allowed metrics). The 'ALL' value means that every metrics will be ignored.
+
+      These properties can be overridden with the previous extension point like this:
+
+      <graphiteReporter enabled="true" periodInSecond="20" host="graphite.server.name" port="2003"
+        prefix="servers.${hostname}.nuxeo.">
+        <allowedMetrics>
+          <metric>nuxeo.cache.user-entry-cache</metric>
+          <metric>nuxeo.cache.group-entry-cache</metric>
+        </allowedMetrics>
+        <deniedMetrics>
+          <metric>nuxeo.cache</metric>
+          <metric>nuxeo.directories</metric>
+        </deniedMetrics>
+      </graphiteReporter>
```

Comment:
ok another syntax that don't require to explain the logic could be an ordered list:
<prefixFilter>
   <allow>nuxeo.cache.user-entry-cache</allow>
   <deny>nuxeo.cache</deny>
   <allow>ALL</allow>
</prefixFilter>

Reply:
True, but with the first syntax it is more similar that the properties that you can set in _nuxeo.conf_

Reply:
it makes sense ok